src/db.c

```diff
@@ -72,8 +72,8 @@ void updateLFU(robj *val) {
  * Even if the key expiry is master-driven, we can correctly report a key is
  * expired on replicas even if the master is lagging expiring our key via DELs
  * in the replication link. */
-robj *lookupKey(redisDb *db, robj *key, int flags, dictEntry *de) {
-    if (!de) de = dbFind(db, key->ptr);
+robj *lookupKey(redisDb *db, robj *key, int flags, dictEntry **deref) {
```

Comment:
please document the output arg.

Reply:
addressed in:
https://github.com/redis/redis/pull/13505/commits/77cf67a1d012f45c7db11184eddea50b69d07fe9

please revise